symfony/messenger/4.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MESSENGER_ADAPTER_DSN": "amqp://guest:guest@localhost:5672/%2f/messages"
```

Comment:
Should we really provide this env var value by default? If there is no need for an adapter at first, can't we set it in the recipe to have `null` as default and add a message explaining how to tweak it in order to use the amqp adapter?

I.e:

```yml
# config/packages/messenger.yaml
parameters:
    env(MESSENGER_ADAPTER_DSN): null
```

or commenting `adapters.default: "%env(MESSENGER_ADAPTER_DSN)%"` 
or providing the `MESSENGER_ADAPTER_DSN="amqp://guest:guest@localhost:5672/%2f/messages"` var commented by default, as in https://github.com/symfony/recipes/blob/bd247214b647c7974bb2e21ac1be8854fdbfdc47/symfony/framework-bundle/3.3/manifest.json#L17.

Reply:
I quite like the idea of commented lines for the adapters and environment variable as well. I've updated the PR accordingly 👍